server/src/test/java/org/elasticsearch/action/admin/indices/template/put/PutIndexTemplateRequestTests.java

```diff
@@ -57,67 +53,6 @@ public void testValidateErrorMessage() {
         assertThat(noError, is(nullValue()));
     }
 
-    public void testMappingKeyedByType() throws IOException {
```

Comment:
This test still seems relevant, since we kept the type wrapping logic in `PutIndexTemplateRequest`:

```
if (source.size() != 1 || source.containsKey(MapperService.SINGLE_MAPPING_NAME) == false) {
    Map.of(MapperService.SINGLE_MAPPING_NAME, source);
}
```

We could wait to remove it until we stop wrapping mappings with a type name?

Reply:
Huh, github dropping my comments apparently...

I reinstated this test to check that mappings wrapped with `_doc` and unwrapped mappings resolve to the same object.